canonical/ubuntu.com
# Implement CVE detailed search

## Done

- Implemented detailed view option for CVE search
- Shows the affected packages and version statuses
- Shows the 5 latest supported releases by default unless versions are filtered, then only selected versions are shown in the table
- Figma: https://www.figma.com/design/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?node-id=949-22651&t=Lj6I7nL7tdPWioks-0

## QA

- Go to https://ubuntu-com-13923.demos.haus/security/cves?q=
- Toggle `Detailed View` slider, see detailed view table
-  Go to `/security/cves?q=2020`, expand the table by clicking `Show all 113 packages` and collapse the table. See that it works as expected
- Filter Ubuntu versions and check that the table only shows selected versions

## Issue / Card

Fixes [WD-11717](https://warthogs.atlassian.net/browse/WD-11717)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11717]: https://warthogs.atlassian.net/browse/WD-11717?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
